std_msgs/Header header
float32[] data
uint32 dim1
uint32 dim2
uint32 dim3